CRM/Contactlayout/Form/RelationshipSelection.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+/**
+ * Relationship Selection Form controller class
+ */
+class CRM_Contactlayout_Form_RelationshipSelection extends CRM_Core_Form {
+
+  /**
+   * Adds the relationship selection field and the save and cancel button.
+   *
+   * @inheritDoc
```

Comment:
Since the function description does not match the parent class then we don't need `{@inheritDoc}` anymore.